.github/workflows/release.yaml

```diff
@@ -143,6 +143,43 @@ on:
           passed as same-named input to release-notes action. Controls whether copies of
           release-notes of sub-components are included.
 
+      categories:
+        type: string
+        default: ''
+        description: |
+          comma separated list of release-note categories to include in slack posting.
```

Comment:
I think it would be nice to mention all categories. for readability, I think it will be preferable to allow linebreak-separates values.

Also, it would be easier to understand the offerered contract by explicitly mentioning it can be configured to select a subset of categories.

Also, `release`-action does not only post release-notes to slack + it accepts a wide range of inputs. By convention (in other such cases), we prefix parameters that are passed to other actions (in this case, e.g. with `release-notes-<name>`) - I think we should keep this pattern.